swdc

```diff
@@ -66,13 +66,43 @@ function applyFixture()
   docker-compose run --rm cli mysql -h mysql -u root -proot $SHOPWARE_PROJECT < "${DIR}/fixtures/${FIXTURE_NAME}.sql"
 }
 
+function isComposerProject()
+{
+    if [ -f "$SHOPWARE_FOLDER/app/install.sh" ]; then
+        echo "true"
+    fi
+}
+
+function buildComposerProjectEnvFile()
+{
+  echo 'SHOPWARE_ENV="development"' > "$SHOPWARE_FOLDER/.env"
```

Comment:
That is not needed.It will be set from .env file

Reply:
Which env file?

Reply:
nevermind i had set this globally to the image https://github.com/shyim/shopware-docker/blob/master/nginx/72/Dockerfile#L3 :)

Reply:
Fine for me. Thanks for the hint